File: C Programming/problem on numbericals/problem83.c
/*

83. Accept N numbers from user and return the difference between largest and smallest number. 

	Input :  N :   6 
	Elements : 85 66 3 66 93 88  
	Output : 90 (93 -3) 

	Author : Milind Nalawade
	
	Date : 11/07/2021
	
	Language : C programming
	
*/

#include<stdio.h>
#include<stdlib.h>
//Logic

int Diff(int* p,int size)
{
	int max=0;
	int min=p[1];
	for(int i=0;i<size;i++)
	{
		if(min==max)
		{
			return 0;
		}
		if(max<p[i])
		{
			max=p[i];
		}
		if(min>p[i])
		{
			min=p[i];
		}
	}
	return max-min;
}

//Entry point function	

int main()
{
	int size=0,ret=0;
	int* p=NULL;
	p=(int*)malloc(sizeof(int));
	printf("how many numbers to take as input : ");
	scanf("%d",&size);
	for(int i=0;i<size;i++)
	{
		printf("\n");
		printf("Enter number at position %d : ",i);
		scanf("%d",&p[i]);
	}
	ret=Diff(p,size);
	printf("\n the differenc betweem maximum and minimum number is : %d",ret);
	printf("\n");
}

File: Python Programming/problems on numbericals/problem83.py

'''
83. Accept N numbers from user and return the difference between largest and smallest number. 
	
	Input :  N :   6 
	Elements : 85 66 3 66 93 88  
	Output : 90 (93 -3)  
	
	Author : Milind Nalawade
	
	Date : 11/07/2021
	
	Language : Python 3.9.0
	
'''
#====
#Logic
#====
def Output(arr):
	max=0
	min=arr[1]
	for i in range(len(arr)):
		if max<arr[i]:
			max=arr[i]
		if min>arr[i]:
			min=arr[i]
	return max-min
	
#=============
#Entry point function
#=============
def main():
	
	arr=[]
	size=int(input("How many element to accept for input : "))
	print()
	print(" Enter {} elements ".format(size))
	
	for i in range(size):
		print()
		arr.append(int(input(" Enter element number {} :".format(i))))
	
	print()
	print("  The difference between maximum and minimum numbers is : {} ".format(Output(arr)))
	print()
#=====
#starter
#=====
if __name__=="__main__":
	main()